storybookjs/storybook
# Upgraded webpack PostCSS plugin definition

Issue: #1061 No PostCSS config found 

## What I did
It seems like either the config is defined differently in Webpack 2 or we upgraded to a newer version of PostCSS. PostCSS can't read our current plugin definition so I upgraded it to the latest listed on the docs.

## How to test
1. create a story, then create a css file for the story.
2. put some test styles in the css.
3. in the story, either `import 'styles.css'`, or `require('styles.css')`.

it should not throw an error during build and console log the style-loader contents.
